tests/wpt/web-platform-tests/XMLHttpRequest/responseurl.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html>
+  <head>
+    <title>XMLHttpRequest: responseURL test</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <link rel="help" href="https://xhr.spec.whatwg.org/#the-responseurl-attribute"/>
+  </head>
+  <body>
+    <div id="log"></div>
+    <script>
+      test(function() {
+        var client = new XMLHttpRequest()
+        assert_equals(client.responseURL, "")
+
+        client.open("GET", "foo.html", false)
+        client.send()
+
+        expected = location.href.replace(/[^/]*$/, 'foo.html')
+        assert_equals(client.status, 404)
+        assert_equals(client.responseURL, expected)
+      })
+      test(function() {
+        var client = new XMLHttpRequest()
+        assert_equals(client.responseURL, "")
+
+        target = "image.gif"
```

Comment:
It was a relative path issue, the `redirect.py` uses a base path of `resources`, so I have to change this from `resources/image.gif` to `image.gif`